features/configuration/home_directory.feature

```diff
@@ -43,24 +43,6 @@ Feature: Configure the home directory to be used with aruba
     The default value is "."
     """
 
-  Scenario: Set to aruba's working directory
-    Given a file named "features/support/aruba.rb" with:
```

Comment:
Because of the final test: testing if the path begins with "/home", which makes it no different from the other two tests (where the paths are fixed strings). On my setup, /home is symlinked, so this doesn't work anyway. I'm not sure how this method is testing the home directory method.